web/sites/default/settings.php

```diff
@@ -53,16 +53,33 @@
   $config['search_api.server.solr']['name'] = 'Lagoon Solr - Environment: ' . getenv('LAGOON_PROJECT');
 }
 
-### Lagoon Redis connection.
-if (getenv('LAGOON')){
-  $settings['redis.connection']['interface'] = 'PhpRedis';
-  $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
-
-  $settings['cache_prefix']['default'] = getenv('LAGOON_PROJECT') . '_' . getenv('LAGOON_GIT_SAFE_BRANCH');
+// Redis configuration.
+if (getenv('LAGOON')) {
+  $redis = new \Redis();
+  $redis_host = getenv('REDIS_HOST') ?: 'redis';
+  $redis_port = getenv('REDIS_SERVICE_PORT') ?: 6379;
```

Comment:
```suggestion
  $redis_port = getenv('REDIS_PORT') ?: 6379;
```

I like this for the env var a lot better.